src/less/textbox/ds6/textbox-base.less

```diff
@@ -18,9 +18,8 @@ textarea.textbox__control {
     box-sizing: border-box;
     color: @ds6-color-g206-gray;
     font-family: inherit;
-    font-size: inherit;
+    font-size: 16px; // for Market Sans to align with buttons and select, it needs same font-size
```

Comment:
Do we want to use the `@ds6-font-size-16` variable here?

https://github.com/eBay/skin/blob/master/src/less/less/ds6/variables.less#L55

Reply:
Yes, will change.

Reply:
Hmm, actually, select and button are also hardcoded to 16px. Seems like they should be rem based to support user defined font-size, but I will revisit this later to make sure I don't cause any regressions for now.